provider/storage_benchmark.py

```diff
@@ -46,31 +48,53 @@ class StorageBenchmark(object):
                         '_kill_pid': 'TASKKILL /F /IM %s /T',
                         '_rm_file': 'RD /S /Q "{}"'}}
     tar_map = {'.tar': '-xvf', '.tar.gz': '-xzf',
-               '.tar.bz2': '-xjf', '.tar.Z': '-xZf'}
+               '.tar.bz2': '-xzf', '.tar.Z': '-xZf'}
     download_cmds = {GIT_DOWNLOAD: 'rm -rf {0} && git clone {1} {0}',
                      CURL_DOWNLOAD: 'curl -o {0} {1}'}
     unpack_cmds = {TAR_UNPACK: '_tar_unpack_file'}
 
-    def __init__(self, os_type, session, name):
+    def __init__(self, os_type, vm, name):
         """
-        :param session: session connected guest
+        :param vm: vm object
+        :type vm: qemu_vm.VM object
         :param name: the name of benchmark
         :type name: str
         """
-        self.session = session
+        self.vm = vm
         self.name = name
         self.os_type = os_type
         self.env_files = []
+        self.session = self._create_session()
 
     def __getattr__(self, item):
         try:
             return self.cmds[self.os_type][item]
         except KeyError as e:
             raise AttributeError(str(e))
 
+    def _create_session(self):
```

Comment:
Actually, this function does not need to return.

Reply:
Thanks, have updated.